dagshub/upload/wrapper.py

```diff
@@ -214,17 +298,43 @@ def _set_default_branch(self):
 
     @property
     def full_name(self):
+        """
+        The full_name function returns the full name of a repository, meaning: "<owner>/<repo>"
+        
+        
+        :return: The full name of a repository
+        
+        """
+  
         return f"{self.owner}/{self.name}"
 
 
 class DataSet:
     def __init__(self, repo: Repo, directory: str):
+        """
+        
+        :param repo (Repo object): Pass a repo object
+        :param directory (str): Specify the directory of the repository
+        :return: A DataSet object
+        
+        """
+
         self.files = {}
         self.repo = repo
         self.directory = self._clean_directory_name(directory)
         self.request_url = self.repo.get_request_url(directory)
 
     def add(self, file: Union[str, IOBase], path=None):
+        """
+        The add function adds a file to the list of files that will be uploaded.
+        
+        
+        :param file (str): Specify the file to be uploaded
+        :param path (str): Specify the path to upload the file
+        :return: Nothing
```

Comment:
Just for consistency
```suggestion
        :return: None
```